tools/palletjack2salt

```diff
@@ -48,7 +48,10 @@ PalletJack2Salt.run do
 
     yaml_output['system'] = system['system']
 
-    yaml_output['service'] = { 'syslog' => system['net.service.syslog'] }
+    yaml_output['service'] = {
+      'syslog' => system['net.service.syslog'],
+      'zabbix' => system['net.service.zabbix']
+    }
```

Comment:
Don't hardcode each service type; publish the entire `net.service` subtree instead.